src/Donations/resources/components/DonationsListTable.tsx

```diff
@@ -252,19 +252,19 @@ export default function DonationsListTable() {
         >
```

Comment:
Let's generate some aria attributes for the anchor & button elements here.

Reply:
Added aria attributes on 3feb90ef5d423da0c266ab7ac57a9c5a5347c6fb.